prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    // Your code here
+    var $container = document.querySelector('#container');
+    console.log($container);
+
+    var $heading = document.createElement('h1');
+    $heading = document.createTextNode('Hello World!');
```

Comment:
Here you are replacing your h1 element with a text node. try to put the text inside the h1 and then append the h1 element to the container div.